blog/content/post/2019-10-02-psec-2019.md

```diff
@@ -0,0 +1,161 @@
+---
+title: 'Platform Security Summit 2019 notes. Part 1'
+abstract: 'We are happy to announce that 3mdeb representation took part in
+          Platform Security Summit 2019. In the next weeks we are going to
+          briefly refer what took our attention and enclose the course of
+          this conference.'
+cover: /covers/PSEC_2019.png
+author:
+  - piotr.krol
+  - lukasz.wcislo
+layout: post
+published: false
+date: 2019-10-03
+archives: "2019"
+
+tags:
+  - PSEC 2019
+  - Conference
+categories:
+  - Miscellaneous
+  - Security
+
+---
+
+PSEC 2019 enables hardware/firmware engineers, VMM/OS developers, architects,
+integrators, verifiers and senior technical staff to collaborate
+on hardware-assisted platform security and composable software supply chain
+integrity, from edge to cloud. Piotr Król, 3mdeb founder and Embedded Systems
+Consultant was one of the speakers with [the article](https://www.platformsecuritysummit.com/#krol) "Less-Insecure Network Edge
+Virtualization with Low Size, Weight and Power" and shared with us his thoughts
+and notes, which will be published here in the next weeks.
+
+![Trespassers will be eaten](/img/PSEC_2019_1.jpg)
+
+The event started 1 Oct in Redmond, WA, at Microsoft Studio B with 3
+introductional videos which were meant to bring the participants into more
+exalted mood. The most interesting of the videos touched the areas of necessity
+of support for organizations like NASA that rise the level of knowledge in all
+the mankind. It also related to the power of stories, myths, that push our
+specie growth. Next all the presentations took place as planned. We'll gonna
```

Comment:
`Species` - it is both singular and plural. Also `specie` means money held in coins rather than notes